server/test/unit/server/webservices/views/serializers/test_serializers.py

```diff
@@ -279,6 +279,109 @@ def test_get_href(self, m_reverse):
             'repo_importer_resource', kwargs={'repo_id': 'apple', 'importer_id': 'pear'})
 
 
+@mock.patch('pulp.server.webservices.views.serializers.ModelSerializer._translate__id')
+class TestTranslateFilters(unittest.TestCase):
+    """
+    Test the translation of arbitrary Mongo queries from external API representation to new
+    internal database fields.
+    """
+
+    def setUp(self):
+        """
+        Creates and configures a model and serializer for all following tests.
+        """
+
+        class FakeSerializer(serializers.ModelSerializer):
+            """Create a fake serialzer for the tests."""
+
+            class Meta:
+                """Defines how fields should be remapped."""
+                remapped_fields = {'internal': 'external', 'id': '_id'}
+
+            def translate_field(self, *args, **kwargs):
+                """Overrides parent to test only the unit."""
+                return 'translated'
+
+        self.serializer = FakeSerializer()
+        self.mock_doc = mock.MagicMock()
+        self.mock_doc._fields = ['internal', 'do_not_change']
+        self.trans_value = 'the key for this value should be translated'
+        self.untrans_value = 'the key for this value should NOT have been translated'
+
+    def test_simple_key_value(self, mock_trans_id):
+        """Test translation of form `{key: value}`."""
+        to_translate = {'external': self.trans_value, 'do_not_change': self.untrans_value}
+        result = self.serializer.translate_filters(self.mock_doc, to_translate)
+        self.assertTrue('translated' in result)
+        self.assertTrue(result['translated'] is self.trans_value)
+        self.assertTrue('do_not_change' in result)
+        self.assertTrue(result['do_not_change']is self.untrans_value)
```

Comment:
I'm surprised that flake8 is happy about the lack of a space between `result['do_not_change']` and `is`.